### PR TITLE
Win oeedgr8r

### PR DESCRIPTION
**Pull request by yakman2020**
Allows windows to compile oeedgr8r without requiring cygwin. mingw64 or other auxilliary issues.

### CODE REVIEW

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 1]
```diff
@@ -14,6 +14,129 @@ then enclave binaries. That information, along with a step-by-step tutorial
 on building an enclave application, is covered in [Getting Started with
 Open Enclave](GettingStarted.md) document.
 
+Using an ACC VM via oe-engine
+------------------------------
+
+The quickest way to begin development would be to use a pre-deployed acc VM. This will install and configure the environment. This document describes performing the deployment on linux. It can also be done on Windows in a similar fashion.
+
+In order to do so, you need to find a binary or build oe-engine, which is located [here](https://github.com/Microsoft/oe-engine).
```
Comment (CodeMonkeyLeet) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: Please split this file out from this PR; @johnkord is setting up a private git repo for docs targeted at MS maintainers.

Comment (anakrish) [reply docs/GettingStartedDocs/GettingStarted.Windows.md]: I get a 404 when accessing https://github.com/Microsoft/oe-engine

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 2]
```diff
@@ -14,6 +14,129 @@ then enclave binaries. That information, along with a step-by-step tutorial
...
+cd into the repository and make the build directory.
+```
+cd openenclave
+mkdir build
+cd build
+
+```
+then cmake. 
+```
+cmake -G "Visual Studio 15 2017 Win64" ..
```
Comment (andschwa) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: So this hard-codes it to a particular version of Visual Studio. You can instead leave it up to the user (or the defaults) and specify a toolset e.g. `-T host=x64` which which will instead pick up the 64-bit toolset for whatever generator is used.

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 3]
```diff
@@ -14,6 +14,129 @@ then enclave binaries. That information, along with a step-by-step tutorial
...
+$env:VC150COMNTOOLS/launchdevcmd.bat
+```
+and run  powershell again. This sets the environment. There is also a batch file to just set the environment, but we use this method.  again, get back to the build directory which we got switched off of.
+
+```
+cd ~/Projects/openenclave/build
+```
+and build
+```
+msbuild ./ALL_BUILD.vcxproj -p:Configuration=Debug
```
Comment (andschwa) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: It's easier to just ignore `msbuild` entirely and call `cmake --build .` which abstracts over whichover toolchain you're using (Make or Ninja or msbuild etc.)

**Comment by CodeMonkeyLeet** [comment]
Superseded by #966 and the documentation has been moved to an internal wiki.

**Pull request closed without merging**